db/migrate/20230428213431_add_foreign_key_constraint_to_active_storage_attachments_for_blob_id.active_storage.rb

```diff
@@ -0,0 +1,10 @@
+# This migration comes from active_storage (originally 20180723000244)
+class AddForeignKeyConstraintToActiveStorageAttachmentsForBlobId < ActiveRecord::Migration[6.0]
+  def up
+    return if foreign_key_exists?(:active_storage_attachments, column: :blob_id)
+
+    if table_exists?(:active_storage_blobs)
+      add_foreign_key :active_storage_attachments, :active_storage_blobs, column: :blob_id
+    end
+  end
+end
```

Comment:
Do we need this migration since we're using `carrierwave` to handle uploads?

Reply:
But let's keep it to avoid less conflicts in the future.

Reply:
it's a no-op, but I wanted to keep all autogenerated files from the update task for consistency